### ISSUE TITLE
[DOC-FIX] Incorrect indentation in Deep Learning Quickstart Step 6

### ISSUE THREAD

**Issue by bartosz-grabowski** [mlflow/mlflow#20514]
### Willingness to contribute

Yes. I can contribute a documentation fix independently.

### URL(s) with the issue

https://mlflow.org/docs/latest/ml/getting-started/deep-learning/#step-6-load-back-the-model-and-run-inference

### Description of proposal (what needs changing)

In the Deep Learning Getting Started guide, Step 6 ("Load back the model and run inference") contains an indentation error in the inference loop. This results in inference being run only on the final batch rather than the full test set.

How it is right now:
```
for data, target in test_loader:
    data, target = data.to(device), target.to(device)
output = model(data)
loss = loss_fn(output, target)

test_loss += loss.item()
_, predicted = output.max(1)
test_total += target.size(0)
test_correct += predicted.eq(target).sum().item()
```

How it should be:
```
for data, target in test_loader:
    data, target = data.to(device), target.to(device)
    output = model(data)
    loss = loss_fn(output, target)

    test_loss += loss.item()
    _, predicted = output.max(1)
    test_total += target.size(0)
    test_correct += predicted.eq(target).sum().item()
```

**Comment by B-Step62** [comment]
@bartosz-grabowski Thanks for reporting the bug! Would you be interested in filing a fix PR?

**Comment by bartosz-grabowski** [comment]
@B-Step62 I opened a fix PR here: https://github.com/mlflow/mlflow/pull/20548.
